tools/test_branch.sh

```diff
@@ -29,6 +31,14 @@ else
     branch=$(git rev-parse HEAD)
 fi
 
+opts_base="$2"
+
+if [[ $# -lt 3 ]]; then
+    opts_branch="$opts_base"
+else
+    opts_branch="$3"
+fi
```

Comment:
This can be written as `opts_branch=${3-$opts_base}`

Reply:
I didn't know this syntax ([here is the doc](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02)), but I think the conditional is easier to read.